jetty-core/jetty-http/src/main/java/org/eclipse/jetty/http/HttpContentWrapper.java

```diff
@@ -110,20 +122,26 @@ public Resource getResource()
     }
 
     @Override
-    public Map<CompressedContentFormat, ? extends HttpContent> getPrecompressedContents()
+    public ByteBuffer getByteBuffer()
```

Comment:
Missing override for `getBytesOccupied()`.

Reply:
fixed